docs/installation.rst

```diff
@@ -0,0 +1,94 @@
+Installation
+============
+
+Installing GeoCAT-viz via Conda
+-------------------------------
+
+The easiest way to install GeoCAT-viz is using
```

Comment:
I'm not 100% on this being how to install geocat-viz via conda, so definitely open to the possibility I'm wrong here, but on the Anaconda website the command to install geocat-viz as a package is `conda install -c ncar geocat-viz`. If the current method listed works, then this comment can be ignored or maybe added to another section on this page but I wanted to make sure this was addressed just in case!

Reply:
I clarified the language so that it says how to either make a new conda environment or just install it on an existing one!